src/app/models/post.ts

```diff
@@ -0,0 +1,6 @@
+export interface Post {
```

Comment:
These models are just used for TypeScript so we know what types we are getting back.  These are manually created by looking at the return from our service (```https://jsonplaceholder.typicode.com```).